pkg/image/source.go

```diff
@@ -76,6 +79,8 @@ func ParseSourceScheme(source string) Source {
 		return OciTarballSource
 	case "oci-registry", "registry":
 		return OciRegistrySource
+	case "singularity-image":
```

Comment:
I think we could use a shorter name here such as `singularity` or `sif` (and "image" is redundant here) -- what do you think @tri-adam ?

Reply:
`sif` would likely be my suggestion - as that's what has been adopted by support brought into `containers/image` and hence `podman` which are using `sif` as the 'transport' name.